### ISSUE TITLE
Inbound chat messages have incorrect timestamps

### ISSUE THREAD

**Issue by placer14** [OpenBazaar/openbazaar-go#1639]
Reports from @drwasho suggest that inbound chat messages are not reporting the correct timestamp on Android but working on iOS:

*Sender (cloud)*
```
[
    {
        "message": "Yo",
        "messageId": "QmQMwKQ3y2evxNzFEkmTkDn3NzUHbHcLUSfmKxszecvujd",
        "outgoing": true,
        "peerId": "QmXgC6fc99c65UAfJgVH1r4uFDQQ4bSDnNig79KgEwPmQy",
        "read": true,
        "subject": "",
        "timestamp": "2019-06-14T05:42:05.999483846Z"
    }
]
```

*Recipient (Android)*
```
[
  {
    "message": "Yo",
    "messageId": "QmQMwKQ3y2evxNzFEkmTkDn3NzUHbHcLUSfmKxszecvujd",
    "outgoing": false,
    "peerId": "QmVBbGn2yHsmUBtU6f6RAA1rGLrvXBGs6LqrevTBLXfBkR",
    "read": true,
    "subject": "",
    "timestamp": "1969-12-31T23:59:58.717372461Z"
  }
]
```
*Recipient (iOS)*
```
[
  {
    "message": "Yo",
    "messageId": "QmPrbSsDLQVmm3StZhaq5CqQpyCG9iQWrkc2iZPFRD2CJD",
    "outgoing": false,
    "peerId": "QmVBbGn2yHsmUBtU6f6RAA1rGLrvXBGs6LqrevTBLXfBkR",
    "read": false,
    "subject": "",
    "timestamp": "2019-06-14T16:18:56.725423+10:00"
  }
]
```

**Comment by drwasho** [comment]
The mobile server on Android devices returns a `timestamp` field from `GET /ob/chatmessages/:peerId` and `GET /ob/chatconversations`.

Chat messages can be sent from a normal node to the recipient Android mobile server, which will convert it to the incorrect values above.

**Comment by placer14** [comment]
We believe the issue is related to Android build using the 32-bit APIs exposed by the gomobile framework as the issue appears specific to the produced build. Testing a 64-bit build is dependent on updating the react-native deps to a more recent version. The mobile team is attempting to produce a 64bit build to confirm. Will hold on this issue until more information becomes available.

**Comment by placer14** [comment]
Resolved by #1549
